### ISSUE TITLE
Authorize.net Accept Hosted  - listen for void payments and cancel order

### ISSUE THREAD

**Issue by abrookbanks** [cubecart/v6#3235]
Ad above. This can be added to the listener.

**Comment by abrookbanks** [comment]
```
Auth:
x_response_code=1&x_response_reason_code=1&x_response_reason_text=This+transaction+has+been+approved%2E&x_avs_code=Y&x_auth_code=JG28SW&x_trans_id=60213878782&x_method=CC&x_card_type=Visa&x_account_number=XXXX1111&x_first_name=Alistair&x_last_name=Brookbanks&x_company=&x_address=63+Heron+Road&x_city=CAMBRIDGE&x_state=&x_zip=CB241AS&x_country=DE&x_phone=&x_fax=&x_email=al%40brooky%2Ecom&x_invoice_num=230404%2D162621%2D5072&x_description=Payment+for+order+230404%2D162621%2D5072&x_type=auth%5Fonly&x_cust_id=&x_ship_to_first_name=Alistair&x_ship_to_last_name=Brookbanks&x_ship_to_company=CubeCart+Support&x_ship_to_address=xxx&x_ship_to_city=xxx&x_ship_to_state=&x_ship_to_zip=xxx&x_ship_to_country=DE&x_amount=85%2E41&x_tax=0%2E00&x_duty=0%2E00&x_freight=0%2E00&x_tax_exempt=FALSE&x_po_num=&x_MD5_Hash=&x_SHA2_Hash=6344C580EDAF976FA0F28FAD1072D79D729FABCAB3BE4CFB1C187595E6792283F5050B1B34CA1CBCB8BAF3791A7C5E9D1689FE89D8B089C7F13F48A6DDA3D8EE&x_cvv2_resp_code=P&x_cavv_response=2&x_test_request=false&cart_order_id=230404%2D162621%2D5072

Void
x_response_code=1&x_response_reason_code=1&x_response_reason_text=This+transaction+has+been+approved%2E&x_avs_code=P&x_auth_code=JG28SW&x_trans_id=60213878782&x_method=CC&x_card_type=Visa&x_account_number=XXXX1111&x_first_name=&x_last_name=&x_company=&x_address=&x_city=&x_state=&x_zip=CB241AS&x_country=&x_phone=&x_fax=&x_email=&x_invoice_num=230404%2D162621%2D5072&x_description=&x_type=void&x_cust_id=&x_ship_to_first_name=&x_ship_to_last_name=&x_ship_to_company=&x_ship_to_address=&x_ship_to_city=&x_ship_to_state=&x_ship_to_zip=&x_ship_to_country=&x_amount=0%2E00&x_tax=0%2E00&x_duty=0%2E00&x_freight=0%2E00&x_tax_exempt=FALSE&x_po_num=&x_MD5_Hash=&x_SHA2_Hash=&x_cvv2_resp_code=&x_cavv_response=&x_test_request=false
```

Notice `x_type=void`

**Comment by abrookbanks** [comment]
Also add setting to *process* auths (like PayPal).

**Comment by abrookbanks** [comment]
Fixed with 1.1.0.
